docs/source/examples/examples.md

```diff
@@ -20,6 +20,7 @@ ZNE with Cirq: Solving MaxCut with QAOA <maxcut-demo.md>
 ZNE with Cirq: Hamiltonian simulation with Pauli gates<hamiltonians.md>
 ZNE with Cirq: Energy of molecular Hydrogen <molecular_hydrogen.md>
 ZNE with Cirq: Noise scaling methods <scaling.md>
+ZNE with Cirq: 1D Ising Simulation <quantum_simulatio_1D_ising.md>
```

Comment:
Thank you for the comment, I just realized that the correct name of the file is <quantum_simulation_1D_ising.md> with the capital "D". I don't know if python is case sensitive. In case it is, could you please add another comment suggesting an edit with the correct name,  so I don't need to update the pull request from my computer and don't mess with other changes?